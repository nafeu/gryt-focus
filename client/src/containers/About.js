import React from 'react'

export default () => (
  <div>
    <h1>About Us</h1>
    <p>The future of gryt-focus</p>
  </div>
)