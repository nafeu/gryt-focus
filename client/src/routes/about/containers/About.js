import React from 'react'

const About = () => (
  <div>
    <h1>About Us</h1>
    <p>The future of gryt-focus</p>
  </div>
)

export default About
