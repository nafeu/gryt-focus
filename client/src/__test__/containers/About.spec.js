import React from 'react'
import renderer from 'react-test-renderer'
import About from '../../containers/About'

describe("About component", () => {
  it('renders correctly without crashing', () => {
    const tree = renderer.create(
      <About />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
})