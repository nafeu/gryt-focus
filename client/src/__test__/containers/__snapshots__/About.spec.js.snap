// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component renders correctly without crashing 1`] = `
<div>
  <h1>
    About Us
  </h1>
  <p>
    The future of gryt-focus
  </p>
</div>
`;
